keras_preprocessing/image.py

```diff
@@ -475,6 +475,10 @@ def load_img(path, color_mode='rgb', target_size=None,
         ImportError: if PIL is not available.
         ValueError: if interpolation method is not supported.
     """
+    if grayscale is True:
+        warnings.warn('grayscale is deprecated. Please use '
+                      'color_mode = "grayscale"')
+        load_img = 'grayscale'
```

Comment:
I'm not sure if this is how you wanted me to handle this? If not, feel free to let me know!

Reply:
Shouldn't it be color_mode='grayscale'?

Reply:
It absolutely should, thanks for pointing that out